spec/jobs/batch_update_job_spec.rb

```diff
@@ -123,14 +127,11 @@
       end
     end
   end
-
+=begin
```

Comment:
It feels it has my old changes crawled back after squashing the git history? I will double check